manifester/commands.py

```diff
@@ -28,3 +32,24 @@ def get_manifest(manifest_category, allocation_name):
             subscription_data=sub,
         )
     manifester.trigger_manifest_export()
+
+
+@cli.command()
+@click.option("--details", is_flag=True, help="Display full inventory details")
+def inventory(details):
+    """Display the local inventory file's contents."""
+    inv = helpers.load_inventory_file(Path(settings.inventory_path))
+    if not details:
+        logger.info("Displaying local inventory data")
+        click.echo("-" * 38)
```

Comment:
not really a performance thing, but since this is repeated, you could put it into a variable like `dashes = "-" * 38` instead.